options/locale/locale_en-US.ini

```
@@ -122,6 +122,7 @@ pin = Pin
 unpin = Unpin
 
 artifacts = Artifacts
+artifact_delete_confirm = Are you sure?
```

```suggestion
are_you_sure = Are you sure?
```

And move it to line 93